src/App.js

```diff
@@ -882,11 +886,14 @@ class App extends React.Component<Props, State> {
   };
 
   render() {
+    const { isSpecificLatLonProvided } = this.state;
     const isNodeRoute = Boolean(this.props.featureId);
     const isNodeToolbarDisplayed = this.isNodeToolbarDisplayed();
 
     const shouldLocateOnStart =
-      !isNodeRoute && +new Date() - (savedState.map.lastMoveDate || 0) > config.locateTimeout;
+      !isSpecificLatLonProvided &&
+      !isNodeRoute &&
+      +new Date() - (savedState.map.lastMoveDate || 0) > config.locateTimeout;
```

Comment:
Unrelated, can you extract this to another variable as well, kinda hard to read :)

Reply:
I cleaned it up now with a `wasMapRecentlyMoved` variable and also got rid of the `+` and optional `0`